tables/alb_access_log/alb_access_log.go

```diff
@@ -0,0 +1,230 @@
+package alb_access_log
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+)
+
+type AlbAccessLog struct {
+	schema.CommonFields
+
+	ActionsExecuted        string    `json:"actions_executed,omitempty"`
+	ChosenCertArn          string    `json:"chosen_cert_arn,omitempty"`
+	Classification         *string   `json:"classification,omitempty"`
+	ClassificationReason   *string   `json:"classification_reason,omitempty"`
+	ClientIP               string    `json:"client_ip"`
+	ClientPort             int       `json:"client_port"`
+	ConnTraceID            *string   `json:"conn_trace_id,omitempty"`
+	DomainName             string    `json:"domain_name,omitempty"`
+	Elb                    string    `json:"elb"`
+	ElbStatusCode          *int      `json:"elb_status_code,omitempty"`
+	ErrorReason            *string   `json:"error_reason,omitempty"`
+	MatchedRulePriority    int       `json:"matched_rule_priority,omitempty"`
+	ReceivedBytes          *int64    `json:"received_bytes"`
+	RedirectURL            *string   `json:"redirect_url,omitempty"`
+	Request                string    `json:"request"`
```

Comment:
Same question on this, should we stored the 3 parts or have them as separate columns `request_method`, `request_uri`, `server_protocol`/`http_version` ? I think it makes more sense to be 3 columns in our table even though it's written as a single var by the log writer - thoughts @cbruno10 ?

Reply:
@graza-io I agree, I think it's easier for the user to query if separated into 3 columns:
- `request_http_method`
- `request_url`
- `request_http_version`

Reply:
I think in `nginx`/`apache` we agreed on using the naming from `nginx`: `request_method`, `request_uri`, `server_protocol`.

If you're happy to run with different columns names here, that's fine.

Reply:
I think it makes sense in the `nginx` tables (and `apache` tables potentially) for the columns to be named according to the native field names. We can also apply the same guideline of using the native log field name/language in this table too.